toast-tk-agent-api/src/main/java/io/toast/tk/plugin/IAgentPlugin.java

```diff
@@ -0,0 +1,12 @@
+package io.toast.tk.plugin;
+
+import java.util.Collection;
+
+import com.google.inject.Module;
+
+public interface IAgentPlugin {
+
+	public void boot();
+
+	public Collection<? extends Module> getModules();
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove usage of generic wildcard type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1452)